resources/lang/es/navmenu.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+return [
+	'search'			=> 'Buscar',
+	'home'				=> 'Inicio',
+	'local'				=> 'Local',
+	'network'			=> 'Red',
+	'discover'			=> 'Discubrir',
+	'viewMyProfile'		        => 'Ver mi perfil',
+	'myProfile'			=> 'Mi Perfil',
+	'myTimeline'	                => 'Mi cronología',
+	'publicTimeline'	        => 'Cronología pública',
+	'remoteFollow'		        => 'Seguimiento remoto',
+	'settings'			=> 'Configuración',
+	'admin'				=> 'Administración',
+	'logout'			=> 'Cerrar sesión',
+	'directMessages'	        => 'Mensajes directos',
+	'composePost'		        => 'Redactar mensaje',
```

Comment:
Maybe `Crear publicación`, although I am not sure were this translation does in the UI so I don't know if it refers to a media post or not.